utils/utils.jsx

```diff
@@ -1554,3 +1554,14 @@ export function copyToClipboard(e, data) {
     document.execCommand('copy');
     document.body.removeChild(textArea);
 }
+
+export function generateRandomString(length) {
```

Comment:
I think this function would be in tests/helpers/, but I'm 2/5 about it.

Reply:
Ah, I didn't know about `tests/helpers/`. I can move it into `client-test-helper.jsx`

Reply:
Done in https://github.com/mattermost/mattermost-webapp/pull/715/commits/cdae7298f0c73e19ec2e515fddf976cd8253e9de